js_modules/dagster-ui/packages/ui-core/src/runs/RunsRoot.tsx

```diff
@@ -45,7 +45,7 @@ export const RunsRoot = () => {
 
   const [filterTokens, setFilterTokens] = useQueryPersistedRunFilters();
   const filter = runsFilterForSearchTokens(filterTokens);
-  const {queryResult, paginationProps} = usePaginatedRunsTableRuns(filter);
+  const {queryResult, paginationProps} = usePaginatedRunsTableRuns(filter, PAGE_SIZE);
```

Comment:
This is PAGE_SIZE=25

Reply:
I believe this pagination hook is fairly new, so I would bet that the `slice` call at line 206 below can be removed, since the query will be handling the page sizing for us.